admin/base/urls.py

```diff
@@ -22,10 +23,19 @@
                          url(r'^users/',
                              include('admin.users.urls', namespace='users')),
                          url(r'^prereg/', include('admin.pre_reg.urls', namespace='pre_reg')),
-                         url(r'^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
+                         url(r'^accounts/password_reset/$', 'django.contrib.auth.views.password_reset', name='reset_password'),
+                         url(r'^accounts/password_reset/confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
```

Comment:
These regexes (well, the entire url) come directly from django, they're used for the password reset flow